resources_portal/models/material_share_event.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+
+from .material import Material
+from .user import User
+
+
+class MaterialShareEvent(models.Model):
+    class Meta:
+        db_table = "material_share_events"
+        get_latest_by = "created_at"
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    material = models.ForeignKey(
+        Material, blank=False, null=False, on_delete=models.CASCADE, related_name="share_events"
+    )
+
+    # TODO: add possible choices
+    event_type = models.CharField(max_length=255, blank=True, null=True)
+
+    time = models.DateTimeField()
+
+    user = models.ForeignKey(
+        User,
+        blank=False,
+        null=False,
+        on_delete=models.CASCADE,
+        related_name="material_share_events",
```

Comment:
What if we named this field `created_by` and had its related_names be `material_share_events_created`?

Reply:
I like it!

Reply:
wouldn't `material_share_events` be enough? not sure what created adds in terms of context?

Reply:
yes! even better! :taco:

Reply:
also probably should keep owner/user/created_by consistent across models since we have several user properties defined

Reply:
I suggested adding `_created` because there are two foreign relationships between `users` and `material_share_events` so it seems less clear that one of them was created by the user as opposed to being associated with them some other way.

If you both think it's obvious we can drop the `_created`.